website/content/en/docs/setup/going-to-prod/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Going to Production
+weight: 3
+---
+
+This section offers guidance and best practices for infrastructure architects deploying Vector into production environments.
+
+{{< info title="Observability Pipelines" >}}
+To simplify going to production, consider Datadog Observability Pipelines, which offers streamlined setup of Vector, out of box monitoring, and change management. It comes with support and offers expert guidance via Datadog Premier Support Services. Contact your sales rep for more information.
```

Comment:
Do all Datadog customers have sales reps? Otherwise, could we include a link here to get in contact with Datadog?